google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/SchemaAwareStreamWriter.java

```diff
@@ -105,6 +105,7 @@ private SchemaAwareStreamWriter(Builder<T> builder)
     streamWriterBuilder.setLocation(builder.location);
     streamWriterBuilder.setDefaultMissingValueInterpretation(
         builder.defaultMissingValueInterpretation);
+    streamWriterBuilder.setMissingValueInterpretationMap(builder.missingValueInterpretationMap);
```

Comment:
Let's add a e2e test in the it directory?

Reply:
Added